groups/groups.yaml

```diff
@@ -729,6 +729,7 @@ groups:
       - zihongz@google.com
       - bowei@google.com
       - pavithrar@google.com
+      - 587340091677@cloudbuild.gserviceaccount.com
```

Comment:
That is correct, the scripts are not in this repo.

Reply:
@prameshj What is this project?  I have no permissions to describe it.

The typical pattern is to switch to cloud build in the staging project, so that billing is attributed properly (and permissions are automatic).

Reply:
Ok, I will add the cloud build scripts for the staging project and revisit this. Thanks.

Reply:
`587340091677@cloudbuild.gserviceaccount.com` is the GCB SA associated with http://louhi.dev (Louhi service). @prameshj is trying to give this Louhi instance permissions to push to gcr.io/k8s-staging-dns.